web-ui/src/api/member.js

```diff
@@ -47,8 +47,8 @@ export const updateMember = async (member) => {
       method: "put",
       url: memberProfileUrl,
       responseType: "json",
-      data: member,
-      withCredentials: true
+      data: { ...member, id: member.id },
```

Comment:
am I just being thick here? what is the need for the `id: member.id` when you already have `...member`?

Reply:
Thanks for catching that, this was the result of a mixup from someone's PR that changed uuid to id

Reply:
I'll wait to push that up in case you find anything else